pkg/storage/query_test.go

```diff
@@ -68,7 +69,18 @@ func TestFindOptions_ToPluginOptions(t *testing.T) {
 	}
 	po := so.ToPluginOptions("mythings")
 	wantSortDoc := bson.D{
-		bson.E{Key: "_id", Value: -1},
-		bson.E{Key: "name", Value: 1}}
-	assert.Equal(t, wantSortDoc, po.Sort)
+		{"_id", -1},
+		{"name", 1}}
+	require.Equal(t, wantSortDoc, po.Sort)
+}
+
+func TestFoo(t *testing.T) {
+	x := bson.D{
+		{Key: "foo", Value: "bar"},
+		{Key: "a", Value: "b"},
+	}
+
+	result, err := bson.Marshal(x)
+	require.NoError(t, err)
+	fmt.Println(string(result))
```

Comment:
Is this intended to just print out the value?

Reply:
Sorry that wasn't supposed to be committed, I was just trying something out and wrote a test to help me see what it serialized as. 🤦‍♀️